### PR TITLE
Added Release Process

### PR DESCRIPTION
**Pull request by jaiminpanchal27**
## Type of change
- [x] Other

## Description of change
Adding Release Steps and Updated readme to remove command to run single test

### CODE REVIEW

**Comment by bretg** [comment]
There's not a prebid-org browserstack is there?

How about updating the download page?

File: RELEASE_SCHEDULE.md [hunk 1]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
 
 Announcements regarding releases will be made to the #headerbidding-dev channel in subredditadops.slack.com.
 
-# FAQs
+## Release Process
+
+1. Make Sure all browserstack tests are passing. Check the last travis build for master branch. If failing please fix the failing tests or skip the test and create issue if it takes too long to fix the failed unit test.
```
Comment (mkendall07) [inline RELEASE_SCHEDULE.md]: +tag contributor.

File: RELEASE_SCHEDULE.md [hunk 2]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
...
+   Update the package.json version to become the current release. Then commit your changes.
+
+   ```
+   git ci -m "Prebid 1.x.x Release"
+   git push
+   ```
+
+3. Publish Release
+
+   Make sure your Prebid code is clean and up to date
```
Comment (mkendall07) [inline RELEASE_SCHEDULE.md]: What does this step mean? NPM? should include specifics.

**Comment by jaiminpanchal27** [comment]
@bretg We do have Browserstack account for Prebid.org. Browserstack Travis integration is currently using that.
For Download page, @mkendall07 can you tell if that can be added to these steps here.

File: RELEASE_SCHEDULE.md [hunk 3]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
 
 Announcements regarding releases will be made to the #headerbidding-dev channel in subredditadops.slack.com.
 
-# FAQs
+## Release Process
+
+1. Make Sure all browserstack tests are passing. Check the last travis build for master branch. If failing please fix the failing tests or skip the test and create issue if it takes too long to fix the failed unit test.
+
+   #### How to run tests in browserstack
```
Comment (bretg) [inline RELEASE_SCHEDULE.md]: Does the coordinator need to run the tests manually or can they just look at the latest travis build? Row 19 says to check Travis -- do we just re-run the tests anyhow?

File: RELEASE_SCHEDULE.md (hunk 2 above)
Comment (bretg) [reply RELEASE_SCHEDULE.md]: +1 on mkendall07's question

File: RELEASE_SCHEDULE.md [hunk 4]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
...
+
+   Make sure your Prebid code is clean and up to date
+
+4. Create a GitHub release
+
+   Edit the most recent release notes draft and make sure the correct tag is in the dropdown. Click `Publish`. GitHub will create release tag. 
+   
+   Pull these changes locally
+
+5. Update coveralls
```
Comment (bretg) [inline RELEASE_SCHEDULE.md]: Could note that coveralls is a code-coverage tool

File: RELEASE_SCHEDULE.md [hunk 5]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
...
+
+3. Publish Release
+
+   Make sure your Prebid code is clean and up to date
+
+4. Create a GitHub release
+
+   Edit the most recent release notes draft and make sure the correct tag is in the dropdown. Click `Publish`. GitHub will create release tag. 
+   
+   Pull these changes locally
```
Comment (bretg) [inline RELEASE_SCHEDULE.md]: Pull the release notes to your local machine? Unclear why that's needed.

File: RELEASE_SCHEDULE.md [hunk 6]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
...
+   Set the environment variables. You may want to add these to your `~/.bashrc` for convenience.
+
+   ```
+   export BROWSERSTACK_USERNAME="my browserstack username"
+   export BROWSERSTACK_KEY="my browserstack access key"
+   ```
+
+   gulp test --browserstack >> prebid_test.log
+   
+   vim prebid_test.log
```
Comment (bretg) [inline RELEASE_SCHEDULE.md]: What are we looking for at this step?

File: RELEASE_SCHEDULE.md [hunk 7]
```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
...
+
+7. Post Release Steps
+   
+   Update the version
+   Manually edit Prebid's package.json to become "1.x.x-pre" (using the values for the next release). Then commit your changes.
+   ```
+   git commit -m "Increment pre version"
+   git push
+   ```
+
```
Comment (bretg) [inline RELEASE_SCHEDULE.md]: Suggest adding a note about updating the Download page.

File: RELEASE_SCHEDULE.md (hunk 2 above)
Comment (jaiminpanchal27) [reply RELEASE_SCHEDULE.md]: Its just verifying that there are no more merges to master in between the process. Updated the step.

File: RELEASE_SCHEDULE.md (hunk 4 above)
Comment (jaiminpanchal27) [reply RELEASE_SCHEDULE.md]: Done

File: RELEASE_SCHEDULE.md (hunk 3 above)
Comment (jaiminpanchal27) [reply RELEASE_SCHEDULE.md]: Added more details.

File: RELEASE_SCHEDULE.md (hunk 5 above)
Comment (jaiminpanchal27) [reply RELEASE_SCHEDULE.md]: To verifying the tag added automatically by github

File: RELEASE_SCHEDULE.md (hunk 6 above)
Comment (jaiminpanchal27) [reply RELEASE_SCHEDULE.md]: Result of unit tests. Added comment

**Comment by jaiminpanchal27** [comment]
@bretg @mkendall07 Have updated the steps.

**Pull request merged**